workflows/references/main.nf

```diff
@@ -186,19 +187,25 @@ workflow REFERENCES {
     )
     germline_resource_vcf_tbi = TABIX_GERMLINE_RESOURCE.out.tbi
 
+    GFFREAD(
+        input.gff_gffread.map { meta, file ->
+            return file ? [meta, file] : null
+        },
+        []
+    )
 
-    if (tools.contains('gffread')) {
-        GFFREAD(input.gff, [])
-
-        gffread = GFFREAD.out.gtf.map { it[1] }
-        versions = versions.mix(GFFREAD.out.versions)
-    }
+    gff_gtf = input.gtf
+        .mix(GFFREAD.out.gtf)
+        .groupTuple()
+        .map { meta, file ->
+            return file[1] ? [meta, file[1]] : [meta, file]
+        }
 
     if (tools.contains('hisat2')) {
         // TODO: be smarter about input assets
         //   Here we either return an empty channel if we have a splice_sites so that HISAT2_EXTRACTSPLICESITES is not triggered
         //   Or we return the provided gtf so that HISAT2_EXTRACTSPLICESITES is run
-        gtf_hisat2 = input.gtf
```

Comment:
If I'm not wrong the `input.gff` branch is not used ?

Reply:
you're totally right, but I'm keeping it for now, I'll do some clean-up later

Reply:
Okay 😄